lib/bodyScrollLock.js

```diff
@@ -143,6 +144,7 @@
         document.body.style.position = 'fixed';
         document.body.style.top = -scrollY + 'px';
         document.body.style.left = -scrollX + 'px';
+        document.body.style.right = '0px';
```

Comment:
Is not needed set measure unit when value equals of 0

Reply:
Fixed